src/main/index.js

```diff
@@ -1,3 +1,7 @@
+import 'core-js/features/number/is-integer'
```

Comment:
with this, the custom function for the isInteger should be changed again?

Reply:
Yeah!

Reply:
Done!